string name

string x_label
string y_label

float32[] x_data
float32[] y_data
